### PR TITLE
Improve deprecation warning for EclGrid.createRectangular

### PR DESCRIPTION
**Pull request by hnformentin**
**Issue**
Deprecation warning had a typo and I could not understand at first glance.

**Approach**
Rewrite deprecations warning.

### CODE REVIEW

**Comment by ertomatic** [comment]
Can one of the admins verify this patch?

**Comment by hnformentin** [comment]
Blocked by #883  and #882  to have tests passing,

File: python/ecl/grid/ecl_grid.py [hunk 1]
```diff
@@ -282,7 +282,7 @@ def create_rectangular(cls, dims, dV, actnum=None):
 
         warnings.warn(
             "EclGrid.createRectangular is deprecated. "
-            + "Please used the similar method in EclGridGenerator!",
+            + "Please use the similar method: EclGridGenerator.createRectangular!",
```
[the remark this answers is not in the document]
Comment (hnformentin) [reply python/ecl/grid/ecl_grid.py]: Sure! Good observation!

**Review by berland — APPROVED** [review]

**Comment by hnformentin** [comment]
just rebased

**Pull request merged**